OxfordBodleian/Ullendorff/BDLeu10.xml

```diff
@@ -34,59 +37,162 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                ወሥጋሃ፡ ለዓመትከ፡ ወለተ፡<hi rend="rubric">ሰንበት፡ እንጐቻ።</hi>
+                            </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="1r121" to="1r159"/>
+                            <title type="complete" ref="LIT6854Prayer"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric">በስመ<supplied reason="omitted">፡</supplied> አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ፡ ጸሎ</hi>ት፡ 
+                                በእንተ፡ ማዕሠረ፡ አጋንንት፡ አካዕ፡ አካብ<supplied reason="omitted">፡</supplied> ክስብኤል፡ ቤቃ፡ ሴቃ፡ ጼቃ፡ ዮድ፡ አኽያ፡ ሸራኽያ፡ በራኽያ፡ 
+                                ኤልሻዳይ፡ ጸባዖት፡ አዶናይ፡ አማኑኤል፡ እልመክኑን፡
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                መጋኛ፡ ወጉስምት፡ ቡዳ፡ ወትግሪዳ፡ ዛር፡ ወቸነፈር፡ ወእምኵሉ፡ ሕማም፡ አድኅና፡ ለዓመትከ፡ ወለተ፡ <hi rend="rubric">ሰንበት፡ እንጐቻ፡</hi> 
+                                ሥመር፡ እግዚኦ፡ እስመ፡ ኵሉ፡ ይትከሃለከ፡ እስመ፡ አልቦ፡ ነገር፡ ዘይሰአኖ፡ ለእግዚአብሔር።
+                            </explicit>
+                        </msItem>
                     </msContents>
                     
                     <physDesc>
-                        <objectDesc form="Codex">
+                        <objectDesc form="Scroll">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
                                 </support>
                                 <extent>
-                                    <measure unit="strip">3</measure>
+                                    <note>Consists of <measure unit="strips">3</measure> strips sewn together.</note>
                                     <dimensions type="outer" unit="mm">
                                         <height>2000</height>
                                         <width>97</width>
                                     </dimensions>
-                                    <measure type="weight" unit="g"></measure>
+                                    <dimensions type="outer" unit="mm" xml:id="strip1">
+                                        <height>790</height>
+                                        <width>97</width>
+                                    </dimensions>
+                                    <dimensions type="outer" unit="mm" xml:id="strip2">
+                                        <height>790</height>
+                                        <width>97</width>
+                                    </dimensions>
+                                    <dimensions type="outer" unit="mm" xml:id="strip3">
+                                        <height>420</height>
+                                        <width>97</width>
+                                    </dimensions>
+                                    <measure type="weight" unit="g">
```

Comment:
You're right; we don't have the weight measurements. I think in this case it's better to comment out the entire element